generics/helpers/utils.js

```diff
@@ -433,6 +433,29 @@ function checkValidUUID(uuids) {
 function checkIfStringIsNumber(str) {
   return /^[0-9]+$/.test(str);
 }
+/**
+  * Return Gotenberg service connection string
+  * @function
+  * @name getGotenbergConnection
+  * @returns {Array}  returns gotenberg server connection.  
+*/
+
+function getGotenbergConnection() {
+
+  let options = {
+    method: "POST",
+    uri: process.env.GOTENBERG_URL + "/forms/chromium/convert/html",
```

Comment:
@borkarsaish65 new env? please make change in envVariable.js and env sample file if it is missing

Reply:
done